### PR TITLE
tickets/DM-39454: rejuvenate Kafka/telegraf refresh scans

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: applications/sasquatch/charts/telegraf-kafka-consumer/templates/configmap.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ data:
       sasl_mechanism = "SCRAM-SHA-512"
       sasl_password = "$TELEGRAF_PASSWORD"
       sasl_username = "telegraf"
-      topic_refresh_interval = "60s"
+      topic_refresh_interval = {{ default "60s" $value.topicRefreshInterval |  quote }}
```
Comment (afausti) [inline applications/sasquatch/charts/telegraf-kafka-consumer/templates/configmap.yaml:47]: Nice, having that parametrized is good.

File: applications/sasquatch/values-idfdev.yaml [hunk 2]
```diff
@@ -60,6 +64,8 @@ chronograf:
   ingress:
     enabled: true
     hostname: data-dev.lsst.cloud
+  image:
+    tag: "1.10.1"
```
Comment (afausti) [inline applications/sasquatch/values-idfdev.yaml:68]: We should be using 1.9.4 everywhere, not critical for dev, but we don't need this change.

**Review by afausti — APPROVED** [review]
There's a small unrelated change otherwise looks good.

File: applications/sasquatch/values-idfdev.yaml (hunk 2 above)
Comment (athornton) [reply applications/sasquatch/values-idfdev.yaml:68]: OK.  We had had 1.10.1 running in dev, and I didn't want to change it.

**Pull request merged**